### PR TITLE
Fixed general errors

### PR DESCRIPTION
**Pull request by DaceyTom2**
## Cleaned up a significant portion of errors getting console logged
A few errors remain from Design System logging errors

### CODE REVIEW

File: __tests__/pages/400.test.js [hunk 1]
```diff
@@ -4,12 +4,11 @@ import { axe, toHaveNoViolations } from 'jest-axe'
 import Error400 from '../../pages/400'
 
 describe('400', () => {
+  const { container } = render(<Error400 />)
   it('renders without crashing', () => {
-    render(<Error400 />)
     expect(screen.getByText('Error 400')).toBeInTheDocument()
```
[the remark this answers is not in the document]
Comment (DaceyTom2) [reply __tests__/pages/400.test.js:9]: I don't believe it needs to be but @shewood would know better. Thought Sheila?

P.S. I did this to speed up tests so they were less likely to fail

[the remark this answers is not in the document]
Comment (s-laugh) [reply __tests__/pages/400.test.js:9]: I think it's smart, only render it once; just wasn't sure if we should be explicitly using `container.get...` in the expect functions

[the remark this answers is not in the document]
Comment (DaceyTom2) [reply __tests__/pages/400.test.js:9]: I believe jest-axe is using the container and testing-library-react uses its own library for render/screen

**Review by shewood — APPROVED** [review]
This is a big improvement on the reduction of errors in the console.

**Pull request merged**